mmv1/products/redis/Cluster.yaml

```diff
@@ -110,6 +110,20 @@ properties:
     default_value: :TRANSIT_ENCRYPTION_MODE_DISABLED
     immutable: true
     required: false
+  - !ruby/object:Api::Type::Enum
+    name: nodeType
+    description: |
+      The nodeType for the Redis cluster.
+      If not provided, REDIS_HIGHMEM_MEDIUM will be used as default
+    values:
+      - :NODE_TYPE_UNSPECIFIED
+      - :REDIS_SHARED_CORE_NANO
+      - :REDIS_HIGHMEM_MEDIUM
+      - :REDIS_HIGHMEM_XLARGE
+      - :REDIS_STANDARD_SMALL
+    default_value: :REDIS_HIGHMEM_MEDIUM
```

Comment:
we cannot add a new field with a default value within an existing block, as that would be a breaking change. We should set to `default_from_api: true`, which would allow Terraform to default to the API's value. It seems like this would fix some of the tests as well.

Reply:
makes sense, updated